client/src/components/chefs-info/index.tsx

```diff
@@ -1,25 +1,30 @@
 import React, { useEffect } from "react";
```

Comment:
it's better to import FC as the following import now:
`import type {FC} from 'react'`